server/controllers/partyController.js

```diff
@@ -0,0 +1,17 @@
+import PartyModel from '../models/partyModel';
+
+class PartyController {
+  static createParty(req, res) {
+    if (req.error) {
+      return res.status(req.error.status).send({ status: req.error.status, error: req.error.msg });
+    }
```

Comment:
@frankly034 Nice work! You can further improve on your work by storing `req.error` in a variable so as to avoid a lookup every time it is used e.g `const error = req.error;` and then use `error`, what do you think about this?

Reply:
@Lumexralph Thanks. 
I see what you said. Your observation is in line. 
I've implemented the change.